apps/files/src/components/FileEntry/FileEntryActions.vue

```diff
@@ -346,7 +346,7 @@ export default Vue.extend({
 <style lang="scss">
 // Allow right click to define the position of the menu
 // only if defined
-.app-content[style*="mouse-pos-x"] .v-popper__popper {
+.content[style*="mouse-pos-x"] .v-popper__popper {
```

Comment:
Isn't it the same? Do we need root class here at all?

```suggestion
[style*="mouse-pos-x"] .v-popper__popper {
```

Reply:
> Isn't it far less efficient?

Probably it is